e2e/utils/consts.ts

```diff
@@ -8,10 +8,12 @@ const baseUrl = `http://localhost:${frontendPort}` as const;
 const registerTexts = {
   weakPassowrd:
     "A jelszónak legalább 6 karakter hosszúnak kell lennie, és tartalmaznia kell legalább egy kisbetűt, egy nagybetűt és egy számot!",
+  successfulRegistration: "Sikeres regisztráció!",
 } as const;
 
 const loginTexts = {
-  wrongEmailOrPassword: "Felhasználó ezzel az email cím, jelszó párral nem létezik, kérjük próbálja újra!",
+  wrongEmailOrPassword:
+    "Felhasználó ezzel az email cím, jelszó párral nem létezik, kérjük próbálja újra!",
```

Comment:
This is so much better than texts in the steps. Really good practice, since we can easily migrate to another language. 💪 , maybe will it is a good time to revisit every text in the application.

Reply:
we will create ticket for it